idelib/dataset.py

```diff
@@ -1003,6 +1031,19 @@ def __init__(self, parent, subchannelId, name=None, units=('', ''),
         self.color = color
 
 
+    def __repr__(self):
+        try:
+            super().__repr__()
```

Comment:
What's this line doing?  Is it trying to assign something to self._unitsStr?

Reply:
Yeah, the superclass' `__repr__()` generates `self._unitsStr`.

Reply:
seems resolved?